src/Arcus.Testing.Assert/Failure/EqualAssertionException.cs

```diff
@@ -4,11 +4,13 @@
 namespace Arcus.Testing
 {
     /// <summary>
-    /// <para>Represents the exception implementation that gets thrown when an actual result does not matches an expectation.</para>
+    /// <para>Represents the exception implementation that gets thrown when an actual result does not match an expectation.</para>
     /// <para>See also: <see cref="AssertXml"/>, <see cref="AssertJson"/>.</para>
     /// </summary>
     [Serializable]
+#pragma warning disable S3925 // Custom exceptions are serializable by default in modern .NET.
```

Comment:
In that case, why don't we just remove the SerializableAttribute ?

Reply:
Done in https://github.com/arcus-azure/arcus.testing/pull/457/commits/3af4b0157cb297cf25279608133ac8f5bc8b149a